pkg/workloads/console/integration/integration_test.go

```diff
@@ -128,10 +131,19 @@ var _ = Describe("Console", func() {
 				csl.Spec.TimeoutSeconds).To(BeNumerically("==", 7200),
 				"console's timeout does not match template's MaxTimeoutSeconds",
 			)
+
+			// Expect(integration.GetEvents(mgr, namespace)).To(
```

Comment:
I decided to leave these tests in, but commented, as a hopefully more obvious reminder that we should go back and fix this, and so that when we do, we don't have to build the assertions up from scratch again.

Reply:
Let's ticket this and leave a `TODO: <ticket-link>` in the code. It doesn't matter that it's open source, it'll just help people understand that this is deliberate instead of accidental.